tetris.rb

```diff
@@ -20,26 +20,50 @@
 set width: size*gameboard_width
 set height: size*gameboard_height+30
 
+game_over = false
+game_over_tick = -1
 t = 1
 update do
-  begin
-    if t % (10 / game.tetromino.fall_rate) == 0
-      game.tetromino.moved = false
-    end
-  rescue
-    game.tetromino.moved = false
+  if game_over && (t - game_over_tick) == 180 # wait 3 seconds before program end
+    close # closes ruby2d window
```

Comment:
```suggestion

```

Let's not close the window for them, they can do that when they've fully absorbed the crushing defeat.

Reply:
We have to delete more than just this line.